prov/rxm/src/rxm_init.c

```diff
@@ -38,7 +38,7 @@
 int rxm_info_to_core(uint32_t version, const struct fi_info *hints,
 		     struct fi_info *core_info)
 {
-	core_info->caps = FI_MSG;
+	core_info->caps = FI_MSG | FI_RMA | FI_ATOMICS;
```

Comment:
Shall we advertise the FI_RMA and FI_ATOMICS caps to the underlying provider?
Seems, FI_RMA was missed, isn't it?

Reply:
You don't require atomics of the core provider though.  I'm not sure that's needed, but FI_RMA looks like it should be there.  @a-ilango, is RMA set based on the user's hints?  Or is RMA used even for large messages, so it should be requested?

Reply:
Yes, FI_RMA should be required from the core provider since RxM uses it for large messages.

We don't need to require FI_ATOMICS unless the app requires it. In case we don't use core provider's atomics we can remove that as well.

@gladkovdmitry17 Did you take a look at prov/util/src/util_atomic.c? @shefty I remember you were mentioning RxM could implement software atomics using the util code.

Reply:
rxm should be able to use the util atomic support.  This PR doesn't actually do the atomics yet, it just sets up the front end API.  @gladkovdmitry17 marked this as WIP.  I added a new 'work in progress' label, so the PR could be marked with that until it's complete.

Reply:
Ok, thanks!
I got it. I'll take a look into `prov/util/src/util_atomic.c` and will re-implement.
@shefty @a-ilango Thanks for the explanation.

@a-ilango adding of FI_RMA will move to the separate PR.